### ISSUE TITLE
RemoveUnusedTerms mentions invalid scalacOption

### ISSUE THREAD

**Issue by MasseGuillaume** [scalacenter/scalafix#625]
> 'locals/privates' is not a valid choice for '-Ywarn-unused'

```
enable -Ywarn-unused:locals/privates
```

should be:

```scala
scalacOptions ++= Seq(
  "-Ywarn-unused:locals",
  "-Ywarn-unused:privates"
)
```

**Comment by olafurpg** [comment]
I think the slash was intended to be interpreted as an "or", but I can see how this is easy to misinterpret. The slash should be replaced with a comma and it would be accepted by scalac then

```
-Ywarn-unused:locals,privates
```

**Comment by mycaule** [comment]
Hello the typo was still here in 
https://scalacenter.github.io/scalafix/docs/rules/RemoveUnusedTerms

**Comment by gabro** [comment]
@mycaule it's fixed on master, but the new documentation site will go online with the 0.6.0 release. A few more days :)

**Comment by mycaule** [comment]
I ended up using just `Ywarn-unused` in my case.
I don't think `-Ywarn-unused:locals` works.

When you load the build.sbt file it is ok, but when I ran compile I had an error.

**Comment by gabro** [comment]
Ah, I just noticed you fixed it on the old website, so it should be online.
@olafurpg the build on master is failing, so that may be it.

**Comment by olafurpg** [comment]
The current live website was manually published from an old commit in v0.5. A few improvements to the old docs never went live, sadly. The breakage on master is unrelated (and it will be fixed by upgrading sbt-docusaurus).
